### DISCUSSION TITLE
a question about the thermal conductivity tensor

### DISCUSSION THREAD

**Discussion by XiangXing96** [ttadano/alamode#198] [Q&A] [answered]
Hi, everyone,

I am using alamode to calculate the thermal conductivity of 2d materials. After I get results, I am confused about the thermal conductivity tensor. **Are the directions in thermal conductivity tensor along real x, y and z or the lattice vectors, a, b, and c**? 
My 2d material is MoS2. It has the same a and b with a angle of 120 degrees. Correspondingly, the atomic arrangements are the same along a and b, while different along x (zigzag) and y (armchair) directions. In thermal conductivity tensor, kxx and kyy are the same, which seem different from the intuition. Therefore, I write this to ask the question.
Thank you very much.

Best regards

**Comment by ttadano** [comment, marked as answer]
The thermal conductivity tensor shown is in the Cartesian coordinate, $\kappa_{xx}$ and $\kappa_{yy}$.

The thermal conductivity tensor $\kappa$ is defined by the Fourier's law:

$$
\boldsymbol{j} = -\kappa \nabla T.
$$

In the Peierls-Boltzmann theory, we do not consider the phonon Hall effect, so the off-diagonal elements of $\kappa$ are zero:

$$
\begin{pmatrix}
j_x \\
j_y \\
j_z 
\end{pmatrix}
= - 
\begin{pmatrix}
\kappa_{xx} & 0 & 0 \\
0 & \kappa_{yy} & 0 \\
0 & 0 & \kappa_{zz}
\end{pmatrix}
\begin{pmatrix}
\nabla_{x}T \\
\nabla_{y}T \\
\nabla_{z}T
\end{pmatrix}.
$$

In the monolayer MoS2, three-fold rotational symmetry exists along the Cartesian $z$ axis. 

If we set the $\vec{a}$ axis to be the same as $\vec{x}$, we have $\kappa_{aa} = \kappa_{xx}$. 
When $\vec{b}$ axis is chosen to be

$$
\vec{b} = \vec{x}\cos{\theta} + \vec{y}\sin{\theta},
$$

with $\theta=\frac{2}{3}\pi$.

Since the $a$ and $b$ directions are symmetrically equivalent, we should have $\kappa_{aa}=\kappa_{bb}$.

Since the heat flux and temperature gradient transform in the same way as the lattice vector, we have

$$
\begin{align}
j_{b} = j_x \cos{\theta} + j_y \sin{\theta} 
= -(\kappa_{xx}\nabla_x T \cos{\theta} + \kappa_{yy} \nabla_y T \sin{\theta}),
\end{align}
$$

and 

$$
\begin{align}
-\kappa_{bb}\nabla_{b}T = -\kappa_{bb}(\nabla_x T \cos{\theta} + \nabla_y T \sin{\theta}).
\end{align}
$$

Since the above two equations must be equal to each other (according to the Fourier's law), we have

$$
\begin{align}
\kappa_{xx}\nabla_x T \cos{\theta} + \kappa_{yy} \nabla_y T \sin{\theta}
= \kappa_{bb}(\nabla_x T \cos{\theta} + \nabla_y T \sin{\theta}).
\end{align}
$$

Thus, for $(\kappa_{xx}=) \kappa_{aa}=\kappa_{bb}$ to hold, it is necessary to satisfy $\kappa_{yy}=\kappa_{xx}$.


Moreover, when you have $\kappa_{xx}=\kappa_{yy}$, you can show that the thermal conductivity value is the same as $\kappa_{xx}$ in arbitrary directions of the $xy$ plane.

**Reply by XiangXing96** [reply]
Hi, Tadano,

I get it. Thank you very much.

Best regards,
